Cargo.toml

```diff
@@ -16,27 +16,27 @@ readme = "README.md"
 bincode = "1.3"
 either = "1.8"
 fnv = "1.0"
-itertools = ">=0.10, <=0.11"
 lazy_static = "1.4"
+itertools = ">=0.10, <= 0.12"
 nom = "7.1"
-quick-xml = "0.28"
+quick-xml = ">=0.28, <= 0.31"
 regex = "1.7"
 regex-cache = "0.2"
 serde = "1.0"
 serde_derive = "1.0"
-strum = { version = "0.24", features = ["derive"] }
+strum = { version = ">=0.24, <=0.25", features = ["derive"] }
```

Comment:
What's the reasoning behind `<= 0.25` ? 0.25 is currently the latest, and afaik there's no "watch out, next release will be breaking" coming up

Reply:
Because 0.26 is semver-breaking according to how cargo defines version... There's no real way to tell Cargo that we'd like any future compatible version, because all future compatible versions are by definition 0.25.x for any x

Reply:
Ahh fair enough, I forgot about semver (and its 0.x specialties) for a second